pkg/assets/landingpage.js

```diff
@@ -98,7 +113,10 @@ function getSignupState(cbSuccess, cbError) {
 // updates the signup state.
 function updateSignupState() {
   getSignupState(function(data) {
-    if (data.status.ready === true) {
+    if (data.status.verificationRequired) {
```

Comment:
i think this would need to be something like: 
` if (data.status.ready === false && data.status.reason == 'VerificationRequired') {
     hideAll()
     show('state-initiate-phone-verification')
}
`

from the console, it looks like the this is what the state is: 
![image](https://user-images.githubusercontent.com/40298444/93123949-dfe3d880-f696-11ea-90fa-15b2ee5bbcbb.png)

before making this change, all i could see in the UI was the spinner and the text: in the UI it continues to spin saying "your codeready toolchain account is waiting approval" - this was after patching the usersignup to require verification and also approving it. I guess it was falling into the `else` clause at the end.. After changing this line above, i'm able to see the request for the `country code` and `phone number` as seen below: 

![image (1)](https://user-images.githubusercontent.com/40298444/93124205-42d56f80-f697-11ea-863c-6c8ff4991c2d.png)

Reply:
Ok, I guessed the response from the source. Maybe this changed since :-) Updated the PR, see 099b471.